core/types/celo_transaction_signing_forks.go

```diff
@@ -64,6 +64,11 @@ func (c *cel2) txFuncs(tx *Transaction) *txFuncs {
 	switch {
 	case t == LegacyTxType && tx.IsCeloLegacy():
 		return deprecatedTxFuncs
+	case t == DynamicFeeTxType:
```

Comment:
Just for my understanding: we don't strictly need to deprecate this here. it would also work as expected when we keep using the london signer here, right?

Reply:
Yeah I think so, but it seems neater to just hand over to the upstream signer for all non celo transactions from the cel2 hardfork onwards.